src/database/queries.ts

```diff
@@ -100,3 +106,49 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Create a new "vendor" document and at it to the vendors collection. 
+ */
+ export async function createVendor(email: string, name: string) {
+  try {
+    const docRef = await addDoc(vendorCollection, {
+      email: email,
+      name: name
+    }).then(docRef => {
+      console.log(docRef.id);
+    })
+  } catch (e) {
+    console.warn('(createVendor)', e);
+    throw e;
+  }
+}
+
+/**
+ * Update an existing "vendor" document in the vendors collection. 
+ */
+export async function updateVendor(uuid: uuid, email: string, name: string) {
```

Comment:
might be easier to split this query into two separate ones for ease of use (e.g. `updateVendorName` and `updateVendorEmail`)

Reply:
I wanna push back on this -- see this comment I left on Sauhard's PR:
https://github.com/calblueprint/vouchers4veggies/pull/10#discussion_r1001347357

I think we should keep an underlying `update` function but expose setters for specific fields.